### ISSUE TITLE
about part alignment

### ISSUE THREAD

**Issue by ch0511** [PKU-ICST-MIPL/OPAM_TIP2018#11]
When training, is ClassNet fine-tuned by 2 aligned parts together to get one PartNet, or fined-tuned seperately by 2 parts to get Part1Net & Part2Net?

If 2 aligned parts are used to fine-tune the ClassNet together, then why is alignment necessary?

I just can't think through these questions, could you please help me? thank you!^_^

**Comment by mumuhe** [comment]
> When training, is ClassNet fine-tuned by 2 aligned parts together to get one PartNet, or fined-tuned seperately by 2 parts to get Part1Net & Part2Net?
> 
> If 2 aligned parts are used to fine-tune the ClassNet together, then why is alignment necessary?
> 
> I just can't think through these questions, could you please help me? thank you!^_^

@ch0511 Yes, 2 parts are used together to get the PartNet. The alignment is done after getting PartNet by Part Alignment.

**Comment by ch0511** [comment]
Thank you!
